package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "dedent",
-	"version": "1.5.1",
+	"version": "1.5.2",
```

No need 🙂 I'll manage this. If other PRs come in the version might not be 1.5.*.

```suggestion
	"version": "1.5.1",
```